priv/perf/apps/load_test/lib/connection/child_chain.ex

```diff
@@ -0,0 +1,17 @@
+defmodule LoadTest.Connection.ChildChain do
+  @moduledoc """
+  Modele that overrides the Tesla middleware with the url in config.
```

Comment:
```suggestion
  Module that overrides the Tesla middleware with the url in config.
```

also for the rest of the PR